activities/Measure.activity/js/activity.js

```diff
@@ -424,27 +426,24 @@ var app = new Vue({
 					csvContent += i + ';';
 					csvContent += session.mode + ';';
 					csvContent += session.date + ';';
-					if (this.log_interval == 300) {
-						csvContent += "5 " + this.SugarL10n.get("Minutes_other") + ";";
-					}
-					else {
-						csvContent += this.log_interval + " " + this.SugarL10n.get("Second") + ";";
-					}
+
+					csvContent += this.log_interval + ";";
 
 					csvContent += j + ';' + session.data[j] + '\n';
 				}
 				i += 1;
 			}
 
+			var vm = this;
 			var metadata = {
 				mimetype: 'text/plain',
-				title: this.currentenv.user.name + "_Measure.txt",
+				title: "Measure Logging by " + vm.currentenv.user.name + ".txt",
```

Comment:
Should be localized.
Plus, use a parameter {{name}} because name position could depend of the language.